cypress/support/commands.ts

```diff
@@ -78,10 +79,30 @@ declare global {
       verifyStepSelectConnector(): void
       verifyStepChooseRuntimeInput(): void
       verifyStepSelectStrategyAndVerifyStep(): void
+      // https://github.com/jaredpalmer/cypress-image-snapshot
+      matchImageSnapshot(snapshotName?: string, options?: any): void
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9241311)
```suggestion
      matchImageSnapshot(snapshotName?: string, options?: unknown): void
```